.github/workflows/scripts/start-rancher.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+flip() {
+    echo $1
+    exit 1
+}
+
+kubectl get ns | grep -s cattle && flip 'rancher already installed?'
+
+set -exu
+
+TMP="${TMP:-/tmp}"
+
+# `helm repo list` emits error code 1 when there are no repos to list
+set +e
```

Comment:
I don't see `helm repo list` being used here so do we still need to remove the `-e` flag?